crypto/params.c

```diff
@@ -903,3 +903,107 @@ int OSSL_PARAM_locate_set_octet_ptr(const OSSL_PARAM *p, const char *key,
 {
     return OSSL_PARAM_set_octet_ptr(OSSL_PARAM_locate(p, key), val, used_len);
 }
+
+int OSSL_PARAM_optional_set_int(const OSSL_PARAM *p, const char *key, int val)
+{
+    return (p = OSSL_PARAM_locate(p, key)) == NULL
+           || OSSL_PARAM_set_int(p, val);
```

Comment:
Correct indentation would be this...  and it looks kinda nice, don't you think?

``` C
    return (p = OSSL_PARAM_locate(p, key)) == NULL
        || OSSL_PARAM_set_int(p, val);
```

Reply:
Not as nice as the || on the line above :)

I'll fix.

Reply:
and pushed.